### PR TITLE
Publish to Prod

### PR DESCRIPTION
**Pull request by calebjacob**

### CODE REVIEW

File: .github/workflows/deploy-prod.yml [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-
-      - name: Install near-social CLI
-        run: |
-          curl --proto '=https' --tlsv1.2 -LsSf https://github.com/FroVolod/near-social/releases/download/v0.2.5/near-social-v0.2.5-installer.sh | sh
-
-      - name: Deploy widgets
-        run: |
-          which near-social
-          echo $PATH
-          ~/.cargo/bin/near-social deploy "$NEAR_SOCIAL_DEPLOY_ID" sign-as "$NEAR_SOCIAL_ACCOUNT_ID" network-config mainnet sign-with-plaintext-private-key --signer-public-key "$NEAR_SOCIAL_ACCOUNT_PUBLIC_KEY" --signer-private-key "$NEAR_SOCIAL_ACCOUNT_PRIVATE_KEY" send
```
Comment (mpeterdev) [inline .github/workflows/deploy-prod.yml:27]: @milenakrawczyk the new prod deploy workflow does not match the logic that was here. We need to deploy to `NEAR_SOCIAL_DEPLOY_ID` and sign as `NEAR_SOCIAL_ACCOUNT_ID`

Comment (milenakrawczyk) [reply .github/workflows/deploy-prod.yml:27]: @mpeterdev someone merged something incorrectly. I fixed it here: https://github.com/near/near-discovery-components/pull/110/commits/7c23bc63d7941b136b69061059d1c8ba24f74398. We now have too many workflow files: 
image

Comment (milenakrawczyk) [reply .github/workflows/deploy-prod.yml:27]: I suggest to delete deploy-prod.yml and deploy-dev.yml

Comment (milenakrawczyk) [reply .github/workflows/deploy-prod.yml:27]: Good catch, thanks! I fixed it in this PR: https://github.com/near/near-discovery-components/pull/130

**Pull request merged**